### ISSUE TITLE
server version is too old for client

### ISSUE THREAD

**Issue by CircleCode** [tmate-io/tmate#245]
since latest tmux update (from 3.1_c-1 -> 3.2-1), running for example `tmux showenv TERM` throws error `server version is too old for client`

is it possible to update tmate's tmux protocol to 3.2 ?

**Comment by geeksam** [comment]
I am also running afoul of this. Can't source my `.tmux.conf` because of it 😢

**Comment by creativeKoder** [comment]
PLeaSe

**Comment by apolopena** [comment]
Yeah I just updated my tmux from 3.0 a to 4.3a and now this is a problem

**Comment by haydenrou** [comment]
I'm seeing this error when trying to use tmux through tmate as well.

**Comment by AlexYegupov** [comment]
https://github.com/tmux/tmux/issues/2212#issuecomment-628351915

**Comment by forbesmyester** [comment]
Happens on default repo versions on Ubuntu Focal (LTS)

```
$ tmate -V
tmate 2.4.0
[0] fozz@saw:~/Projects
$ tmux -V
tmux 3.2a
```

**Comment by pataquets** [comment]
I'd love to be able to move windows from my private work session to tmate's.

**Comment by pataquets** [comment]
@nviennot Fancy to share current status/intentions/plans on bringing tmate up to date? Thanks!

**Comment by GottZ** [comment]
am I mistaken if I'd say this repo is abandoned?
